pkgs/development/python-modules/bootstrap/build/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, python
+, build
+, flit-core
+, installer
+, packaging
+, pyproject-hooks
+, tomli
+}:
+let
+  buildBootstrapPythonModule = basePackage: attrs: stdenv.mkDerivation ({
+    pname = "${python.libPrefix}-bootstrap-${basePackage.pname}";
+    inherit (basePackage) version src meta;
+
+    buildPhase = ''
+      runHook preBuild
+
+      PYTHONPATH="${flit-core}/${python.sitePackages}" \
+          ${python.interpreter} -m flit_core.wheel
+
+      runHook postBuild
+    '';
+
+    installPhase = ''
+      runHook preInstall
+
+      PYTHONPATH="${installer}/${python.sitePackages}" \
+          ${python.interpreter} -m installer \
+              --destdir "$out" --prefix "" dist/*.whl
```

Comment:
What is the reason destdir is set here to $out instead of prefix?

Reply:
Great catch! This has been fixed in the latest version. I didn't know what the difference was, but it turns out there is an observable difference in the RECORD file, and using prefix is correct.